Changelog.md

```diff
@@ -1,170 +1,174 @@
-## Ignition Launch 6.x
+## Gazebo Launch 6.x
 
-### Ignition Launch 6.X.X (202X-XX-XX)
+### Gazebo Launch 6.X.X (202X-XX-XX)
+1. Migrate `IGN_LAUNCH_PLUGIN_PATH` environment variable to `GZ_LAUNCH_PLUGIN_PATH` for finding
```

Comment:
Oh did you mean to put this into `Migration.md`? We update the changelog right before a release

Reply:
Ah! I think I got confused when you mentioned the changelog :sweat_smile: 

https://github.com/gazebosim/gz-launch/pull/163/commits/feb94cd807ee5fadee22552eb835d064d0d2192c